ibis/expr/datatypes/core.py

```diff
@@ -417,6 +459,36 @@ class Timestamp(Temporal, Parametric):
     scalar = "TimestampScalar"
     column = "TimestampColumn"
 
+    @classmethod
+    def from_unit(cls, unit, timezone=None, nullable=True):
```

Comment:
Now these are not actually used, see the commented out code in `ibis.formats.pyarrow`